src/lib/utils/time.test.ts

```diff
@@ -8,33 +8,30 @@ const dateFromLastYear = new Date(
   currentDate.getDate()
 )
 const twoYearsAgo = new Date(currentDate)
-const twoYearsAhead = new Date(currentDate)
 twoYearsAgo.setFullYear(currentDate.getFullYear() - 2)
+
+const twoYearsAhead = new Date(currentDate)
 twoYearsAhead.setFullYear(currentDate.getFullYear() + 2)
 
 describe(isLastYear, () => {
-  it('returns true for a date from the previous year'),
-    () => {
-      expect(isLastYear(dateFromLastYear)).toBe(true)
-    }
+  it('returns true for a date from the previous year', () => {
+    expect(isLastYear(dateFromLastYear)).toBe(true)
+  })
 
-  it('returns false for a date of the current year'),
-    () => {
-      expect(isLastYear(currentDate)).toBe(false)
-    }
+  it('returns false for a date of the current year', () => {
+    expect(isLastYear(currentDate)).toBe(false)
+  })
 })
 
 describe(categorizeDate, () => {
-  it('returns the year two years ago'),
-    () => {
-      expect(categorizeDate(twoYearsAgo)).toBe(2022)
-    },
-    it('returns "today" '),
-    () => {
+  it('returns the year two years ago', () => {
+    expect(categorizeDate(twoYearsAgo)).toBe(2022)
```

Comment:
The categorizeDate tests need to be changed now that you return an object, right?